### ISSUE TITLE
Odd Request (related to Gamma vs Linear)

### ISSUE THREAD

**Issue by holdingjason** [crest-ocean/crest#97]
Ok this is going to show some severe ignorance on my part but whatever.

So I accidentally left the project in gamma mode vs linear and the waves look quite spectacular.  Anyway we can get the ocean to look like the gamma setup in linear mode (tried just about every combo and just cant get this look?)  Really more of a question on what changes between the two and maybe to fill in my massive knowledge gaps on the issue if you know off the  top of your head.  Going to do some research on my own as well to fill that in but figured what the hell i would ask anyway and get your thoughts.

LOW LOW priority so please feel free to ignore or let me know if I should ask stuff like this directly.

Thanks

**Comment by huwb** [comment]
Oh!!

Thats tough - its non trivial. I'll think about it some more and post back. It might be useful if you could nail down exactly what component is different - a bunch of the features are on toggles so you could turn everything off and then slowly build it up. This could give a hint which might jog my memory/tired brain!

**Comment by holdingjason** [comment]
Thanks.  Ok will play with that and get back to you.

**Comment by teidenzero** [comment]
Hi!
I'm not sure this can be done and I'm not in front of Unity right now but if you can apply post process selectively and only have it affect the ocean, you could use the ColorCorrection and change the curve from linear to gamma manually.

**Comment by holdingjason** [comment]
Thanks for the suggestion.  Unfortunately it appears the post process works on the camera so cannot be applied to specific objects (or layers).  You could probably do it with a separate camera.

**Comment by huwb** [comment]
Yeah my impression/understanding is the same as what Jason said.

Is this still pending on your list Jason? Just checking, bumping some issues to see if I can close some out.

**Comment by holdingjason** [comment]
Go ahead and close it out.  Might experiment more with it at some point.

**Comment by huwb** [comment]
(just changing the title so i can see what it is while searching through the issues)
